nucypher/cli/commands/stake.py

```diff
@@ -451,8 +454,8 @@ def unbond_worker(general_config, transacting_staker_options, config_file, force
 @option_value
 @option_lock_periods
 @group_general_config
-@option_only_lock
-def create(general_config, transacting_staker_options, config_file, force, value, lock_periods, only_lock):
+@option_use_uncollected_rewards
```

Comment:
:+1:  How about "unclaimed" throughout; `--from-unclaimed-rewards`

Reply:
🤔 Interestingly, I had `unclaimed` before, but I realized that we only use "claim" for worklock in our code - "collect" was used in the context of rewards.

I'm fine to change it if desired, but was trying to keep the parlance the same.

Reply:
Using "from" instead of "use" is good

Reply:
✔️